README.md

```diff
@@ -1 +1,42 @@
-# Survey-form
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+##Survey form
```

Comment:
You must add space between ## and survey

Reply:
Please be more polite when requesting a change. This will make the other person not feel like they are being commanded. Even though the change is required.